Bindings/Python/tests/test_basics.py

```diff
@@ -98,5 +98,26 @@ def test_ToyReflexController(self):
         controller = osim.ToyReflexController()
         
     def test_GCVSplineSet(self):
-        splineset = osim.GCVSplineSet(os.path.join(test_dir, 'std_subject01_walk1_ik.mot'))
-        splineset = osim.GCVSplineSet(osim.TimeSeriesTable(os.path.join(test_dir, 'std_subject01_walk1_ik.mot')), [], 5, 0)
+        splineset = osim.GCVSplineSet(os.path.join(test_dir,
+            'std_subject01_walk1_ik.mot'))
+        splineset = osim.GCVSplineSet(
+                osim.TimeSeriesTable(os.path.join(test_dir,
+                    'std_subject01_walk1_ik.mot')), [], 5, 0)
+
+    def test_deserialize_tool_with_empty_model_file(self):
+        # Previously, there was a bug where loading an AbstractTool (e.g.,
```

Comment:
I don't think a historical record is necessary. Perhaps simply "Ensure an exception is thrown if…".